decaylanguage/dec/dec.py

```diff
@@ -452,7 +452,7 @@ def _add_charge_conjugate_decays(self):
         def _is_not_self_conj(t):
             try:
                 mname = t.children[0].children[0].value
-                if Particle.from_dec(mname).is_self_conjugate:
+                if Particle.from_string(mname).is_self_conjugate:
```

Comment:
In the end all `from_dec` are in fact to be replaced by `from_evtgen_name` but there are some cases missing in the `Particle` package, which explains the failures you probably saw in your tests, which made you use `from_string`. I've had since weeks a start of code to fix all this. It's now high on my list so that this package can finally depend on `Particle` 0.8.*. For now let's go with your temporary workaround (indeed `from_string` is not guaranteed to always return the correct match).